tests/test_create.py

```diff
@@ -165,6 +165,9 @@ def test_just_python2(self):
             assert exists(join(prefix, PYTHON_BINARY))
             assert_package_is_installed(prefix, 'python-2')
 
+            remove_from_env(prefix, '--all')
+            assert not exists(prefix)
```

Comment:
Except... This is going to break in master.  Changed the format for `remove_from_env` slightly.  Just `run_command(Commands.REMOVE, prefix, '--all')` now.  Just rebase against master and go again.